src/videojs.ima.js

```diff
@@ -305,7 +305,13 @@
         player.ads.startLinearAdMode();
       }
       adContainerDiv.style.display = 'block';
-      controlsDiv.style.display = 'block';
+      // Don't show ad controls for not video ads (like modal ads)
+      if (adEvent.getAd().getContentType().search(/video/i) !== 0) {
```

Comment:
getContentType function return ads mimetype and video mimetypes must be start with "video" like "video/mp4", "video/webm", "video/ogg". So, "video" string must be have index number 0. We don't want to show controls on wrong mimetypes like "x-flv/video".